docs/_sources/review-submissions.rst.txt

```diff
@@ -0,0 +1,8 @@
+Review Submissions
```

Comment:
question... why is this called "review submissions"? Submissions are just workflow submissions... not dependent on Review

Reply:
My initial feeling was that while submissions dont require review, most of the functionality of that module is around submissions to review and that new users would find "Submissions" too ambiguous. Don't feel strongly though, so changed it

Reply:
gotcha - I see what you mean. I think either submissions or workflow-submissions would totally work.
side note - should this file get deleted since you have a new `submissions.rst` file?

Reply:
sphinx-build doesn't delete old html files automatically- went and manually removed these